types/debug/index.d.ts

```diff
@@ -28,6 +30,7 @@ declare namespace debug {
 
     export interface IDebugger {
         (formatter: any, ...args: any[]): void;
+        (...args: string[]): void;
```

Comment:
Hm... yes, you are right, did not think about that.

There is sadly no way to type an array with a minimum length. This might have to stay in.

Sorry to have bothered you